src/rust/cryptography-x509-verification/src/policy/mod.rs

```diff
@@ -265,13 +269,21 @@ impl<'a, B: CryptoOps + 'a> PolicyDefinition<'a, B> {
     /// **IMPORTANT**: This is **not** the appropriate API for verifying
     /// website (i.e. server) certificates. For that, you **must** use
     /// [`Policy::server`].
-    pub fn client(ops: B, time: asn1::DateTime, max_chain_depth: Option<u8>) -> Self {
+    pub fn client(
+        ops: B,
+        time: asn1::DateTime,
+        max_chain_depth: Option<u8>,
+        ca_extension_policy: Option<ExtensionPolicy<'a, B>>,
+        ee_extension_policy: Option<ExtensionPolicy<'a, B>>,
+    ) -> Self {
```

Comment:
It should be possible to split out making these arguments (but always passing None).